### PR TITLE
Change in selector of apps

### PR DESCRIPTION
**Pull request by germanohn**
Change resolver appsWithStats to show the apps published by a specific vendor

### CODE REVIEW

File: node/resolvers/appsWithStats.ts [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
-    map(removeVersionFromAppId),
-    filter(startsWith(account))
-  )(deps)
-}
\ No newline at end of file
+  const {dataSources: {registry}, vtex: {account}} = ctx
+  const publishedAppsInThisAccount = await registry.listApps()
+    .then(prop('data'))
+    .then(pluck('partialIdentifier'))
+    .then(result => filter(includes(account), result))
```
Comment (diegoximenes) [inline node/resolvers/appsWithStats.ts]: It is unlikely but what if the string account is in an app's name?
Example: `vendor.appvtex` can be interpreted as an app from vendor vtex.

**Review by diegoximenes — CHANGES_REQUESTED** [review]
Nice! 
Only a little comment.
What do you think?

File: node/resolvers/appsWithStats.ts (hunk 1 above)
Comment (germanohn) [reply node/resolvers/appsWithStats.ts]: makes sense

File: node/resolvers/appsWithStats.ts [hunk 2]
```diff
@@ -1,14 +1,27 @@
-import { removeVersionFromAppId } from '@vtex/api'
-import { compose, filter, isEmpty, isNil, keys, map, reject, startsWith } from 'ramda'
+import { filter, includes, map, pluck, prop } from 'ramda'
 
-const nullable = x => isEmpty(x) || isNil(x)
+
+const filterAppNameByAccount = (appNameList: string[], accountName: string) => {
+  const isVendorInApp = (appName: string) => {
+    const appNamePart: string[] = appName.split('.')
+    return includes(accountName, appNamePart[0])
```
Comment (diegoximenes) [inline node/resolvers/appsWithStats.ts]: I think `includes` is not appropriate here. 
It is unlikely, but consider this: `vendorvtexvendor.appName` can be interpreted as an app from vendor `vtex`.

**Review by diegoximenes — CHANGES_REQUESTED** [review]
Almost there!

**Review by diegoximenes — APPROVED** [review]

**Pull request merged**